src/app/shared/components/comment-form/comment-form.component.ts

```diff
@@ -48,44 +49,61 @@ export class CommentFormComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
+    this.updateFormType();
     if (this.data.comment != null) {
```

Comment:
Editing a comment is currently not possible anyway but I didn't want to get rid of this code. It will be useful once we start working on https://github.com/plezanje-net/web/issues/375